### DISCUSSION TITLE
pyscript.app_config missing?

### DISCUSSION THREAD

**Discussion by alistairg** [custom-components/pyscript#386] [Q&A]
I'm running the latest release, installed via HACS.

I must be doing something obvious wrong, but in my code when I try and reference `pyscript.app_config` I get a `NameError: name 'pyscript' is not defined` error.

Help! :)

**Comment by marshalltech81** [comment]
I am getting the same issue and cannot figure it out

**Comment by craigbarratt** [comment]
See #313 and #518.  I also just added some more information to the docs; see 12b1e84.
